lib/broadway_dashboard.ex

```diff
@@ -19,13 +19,16 @@ defmodule BroadwayDashboard do
 
   @impl true
   def init(opts) do
-    pipelines = Keyword.get(opts, :pipelines, [])
+    # In case no pipeline is passed in the router, we auto-discover them.
+    pipelines = with [] <- opts[:pipelines] || [], do: :auto_discover
```

Comment:
I would go with: `opts[:pipelines] || :auto_discover`. So an empty list would be an explicit way to disable `:auto_discover`. Maybe not useful here, because we can always remove the page, but it could be useful for the Ecto one.